pkg/kvstore/allocator/allocator.go

```diff
@@ -681,13 +681,15 @@ func (a *Allocator) Release(ctx context.Context, key AllocatorKey) (lastUse bool
 }
 
 // RunGC scans the kvstore for unused master keys and removes them
-func (a *Allocator) RunGC() error {
+func (a *Allocator) RunGC(keysToDelete map[string]uint64) (map[string]uint64, error) {
```

Comment:
I think `staleKeysPrevRound` would be a better name because we are not necessarily deleting those keys.